examples/multigpu_advtrain/model.py

```diff
@@ -0,0 +1,339 @@
+"""Lightweight model objects in TensorFlow.
+
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from utils import ordered_union
+
+from cleverhans.model import Model
+
+from tensorflow.python.training import moving_averages
+
+
+def clone_variable(name, x, trainable=False):
+    return tf.get_variable(name, shape=x.shape, dtype=x.dtype,
+                           trainable=trainable)
+
+
+class MLP(object):
```

Comment:
If this is straight from the tutorials, should we just import it from the tutorials?

Reply:
Fixed in 714afc0. Now inherits from MLP in tutorial_models.